packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1716,6 +1708,19 @@ class ThemeData with Diagnosticable {
   )
   final AndroidOverscrollIndicator? androidOverscrollIndicator;
 
+  /// The color used to highlight the active states of toggleable widgets like
+  /// [Switch], [Radio], and [Checkbox].
+  Color get toggleableActiveColor => _toggleableActiveColor!;
+  final Color? _toggleableActiveColor;
+
+  /// The color used to highlight selected rows.
+  @Deprecated(
+    'No longer used by the framework, please remove any reference to it. '
+    'This feature was deprecated after v3.1.0-0.0.pre.',
+  )
+  Color get selectedRowColor => _selectedRowColor!;
```

Comment:
Same here, moving things around unnecessarily makes it hard to parse the git history.

Reply:
Moving per our strict organisation of `ThemeData`